doc/uftrace-report.md

```diff
@@ -24,25 +24,26 @@ REPORT OPTIONS
 ==============
 -f *FIELD*, \--output-fields=*FIELD*
 :   Customize field in the output.  Possible values are: `total`, `total-avg`,
-    `total-min`, `total-max`, `self`, `self-avg`, `self-min`, `self-max`, `size`,
-    `call` and `all`.  Multiple fields can be set by using comma.  Special field
+    `total-min`, `total-max`, `total-stdv`, `self`, `self-avg`, `self-min`, `self-max`,
+    `self-stdv`, `size`, `call` and `all`.  Multiple fields can be set by using comma.  Special field
     of 'none' can be used (solely) to hide all fields and 'all' can be used to
     show all fields.
-    Default is 'total,self,call'.  See *FIELDS*.
+    Default is 'total,self,call'. See *FIELDS*. `stdv` stands for standard deviation, it would only contain value
+    if the function is invoked multiple times.
 
 -s *KEYS*[,*KEYS*,...], \--sort=*KEYS*[,*KEYS*,...]
 :   Sort functions by given KEYS.  Multiple KEYS can be given, separated by
     comma (,).  Possible keys are `total` (time), `total-avg`, `total-min`,
-    `total-max`, `self` (time), `self-avg`, `self-min`, `self-max`, `size`,
+    `total-max`, `total-stdv`, `self` (time), `self-avg`, `self-min`, `self-max`, `self-stdv`, `size`,
```

Comment:
Please keep the line length as similar as others.  I think I kept it around 80 characters per line.  It's not the part of the auto formatting but it'd be nice if you follow the convention even if it ends up touching unchanged lines.